src/parquet/column/writer.cc

```diff
@@ -97,9 +100,11 @@ void ColumnWriter::WriteNewPage() {
   }
 
   // TODO(PARQUET-590): Encodings are hard-coded
+  Encoding::type encoding = Encoding::PLAIN;
+  if (has_dictionary_) { encoding = Encoding::PLAIN_DICTIONARY; }
```

Comment:
Add a TODO that this is different in Parquet V2 (where the data pages have `RLE_DICTIONARY` encoding). If we are encoding for V1 writers, then we must use `PLAIN_DICTIONARY` (and the dictionary itself must be written out in plain encoding)